blocks/init/src/Blocks/components/search-bar/search-bar.php

```diff
@@ -38,10 +42,14 @@
 	class="<?php echo \esc_attr($searchClass); ?>"
 	action="<?php echo \esc_url($searchBarAction); ?>"
 >
+	<?php if ($searchBarLabelShow) { ?>
+		<label for="<?php echo \esc_attr($searchBarId); ?>"><?php echo \esc_html($searchBarLabel); ?></label>
+	<?php } ?>
```

Comment:
Arguably `<label>` should be set even if it's not shown (accessibility, screen reader should know what this input is for). But not sure if we need to support this functionality out of the box in the component or does this fall more in the line of "You need to set this properly in your project if you're not showing the label" 🤷‍♂️

Reply:
Definitely should be set out of the box so that we have a11y compliance covered.

By your logic, everything could be set in your project, but if we are exposing certain things in our libs, it's our responsibility to make it accessible, no?

Reply:
>  it's our responsibility to make it accessible, no?

Yeah I agree for sure. Tho then we need to change this so that `<label>` is always rendered but if `$searchBarLabel === false` we just give it a class that hides the visually hides text. (https://infinum.github.io/eightshift-frontend-libs/sassdocs/#placeholder-visually-hidden - `%visually-hidden`)

Reply:
Also then the markup needs to be something like (so we can **visually** hide `.label-text` if needed):

```html
<label>
  <span class="label-text">Some label</span>
  <input />
</label>
```

Reply:
Good suggestion, I'll change this